src/ui/components/Library/RecordingRow.tsx

```diff
@@ -75,7 +75,7 @@ function RecordingRow({
     }
 
     loadReplayPrefs(recording.id);
-    router.push(url);
+    window.location.href = url;
```

Comment:
I wonder if it is any better to use a Next Link element https://nextjs.org/docs/api-reference/next/link

Reply:
I don't think it'll produce the same result. I believe it does client side route change by default. We can investigate more in the follow up.